code/parse/sexp.cpp

```diff
@@ -19804,6 +19805,44 @@ void multi_sexp_set_respawns()
 	}
 }
 
+/**
+ * set a hotkey for one or more ships/wings
+ */
+void sexp_add_remove_hotkey(int node)
+{
+	int objnum, setnum, n = node;
+	bool is_adding; // True for add, False for remove
+	is_adding = is_sexp_true(n);
+	n = CDR(n);
+	setnum = eval_num(n);
```

Comment:
You should test to see whether setnum is in the range of allowable hotkeys.  Also I think the maximum hotkey value is reserved for something special.